### PR TITLE
Update qa code to add a flag in a new database table and update columns

### PR DESCRIPTION
**Pull request by andrewjtdunn**
As requested in 5/17 meeting

### CODE REVIEW

**Comment by andrewjtdunn** [comment]
@abejburton how should I interpret the recommit failing here? It's not immediately clear to me what I should do

**Comment by abejburton** [comment]
@andrew pre commit is changing the line of code that is in red to what is in green because your original code has a line that is too long. If you want that to not happen, you need to run pre-commit on your code before pushing to github. It will automatically edit the line so that it isn't too long. Then add those changes, commit, and push.

**Comment by andrewjtdunn** [comment]
@abejburton in this instance, I don't really think the error is reasonable -- as far as I can tell, it's saying my SQL insertion, contained in triple quotes, is an errant comment. Moreover, when I follow its recommendation and assign that text to a variable, it then flags the variable with an error. Any advice on how to handle that?

**Comment by andrewjtdunn** [comment]
@abejburton in this instance, I don't really think the error is reasonable -- as far as I can tell, it's saying my SQL insertion, contained in triple quotes, is an errant comment. Moreover, when I follow its recommendation and assign that text to a variable, it then flags the variable with an error. Any advice on how to handle that?

EDIT: it seems to have solved itself, I'm not sure how

File: civiclens/collect/move_data_from_api_to_database.py [hunk 1]
```diff
@@ -194,6 +194,38 @@ def verify_database_existence(
...
+    connection, cursor = connect_db_and_get_cursor()
+    with connection:
+        with cursor:
+            cursor.execute(
+                """INSERT INTO regulations_dataqa (
+                    "data_id",
+                    "data_type",
+                    "error_message",
+                    "added_at"
+                ) VALUES (%s, %s, %s, CURRENT_TIMESTAMP)
```
Comment (rezarzky) [inline civiclens/collect/move_data_from_api_to_database.py:218]: should move CURRENT_TIMESTAMP to avoid SQLi

**Review by rezarzky — COMMENTED** [review]
@andrewjtdunn looks good, but try to merge the previous PR. we need to resolve the conflict

**Review by rezarzky — APPROVED** [review]
LGTM

**Pull request merged**